src/modules/common/hooks/useDBConnection.ts

```diff
@@ -13,16 +13,24 @@ export const dataSource = new DataSource({
 })
 
 export const useDBConnection = () => {
+  const [isInitialized, setIsInitialized] = useState(false)
+  const [error, setError] = useState(false)
```

Comment:
suggest using more descriptive name which also will be consistent with string 16
```suggestion
  const [isError, setisError] = useState(false)
```